entity-service-api/src/main/proto/org/hypertrace/entity/type/service/v2/entity_type_service.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+package org.hypertrace.entity.type.service.v2;
+
+message EntityType {
+  string name = 1;
+  string attribute_scope = 2;
+  string id_attribute_key = 3;
+}
+
+message EntityTypeFilter {
+  repeated string name = 1;
+}
+
+message Empty {}
+
+service EntityTypeServiceV2 {
```

Comment:
Can we remove the v2 from the name? Even if it's annoying to fully qualify it in one spot now, it allows us to fully cut over later without another breaking change.

Reply:
Done